BackEndFlask/models/logger.py

```diff
@@ -49,17 +49,32 @@ def __try_clear(self):
         Clears all entries that are older than 90 days.
         """
         now = datetime.now()
+
         for handler in self.logger.handlers:
             if isinstance(handler, logging.FileHandler):
                 with open(handler.baseFilename, 'r+') as f:
                     lines = f.readlines()
+
                     f.seek(0)
+
+                    last_parsed_time: None
```

Comment:
`last_parsed_time: None` -> `last_parsed_time = None`

Reply:
I committed changes to replace the : for a =!